scripts/macpython-install-python.sh

```diff
@@ -405,19 +351,7 @@ function make_workon_venv {
 #echo "Remove and update Python files at ${MACPYTHON_FRAMEWORK}"
 #sudo rm -rf ${MACPYTHON_FRAMEWORK}
 
-if test "$(arch)" == "arm64"; then
-  echo "we are arm"
-  PLAT=arm64
-  for pyversion in $LATEST_3p9 $LATEST_3p10 $LATEST_3p11; do
-    install_macpython $pyversion 11
-    install_virtualenv
-  done
-else
-  # Deployment target requirements:
-  # * 10.9: Python 3.7
-  # * 11: Python >= 3.8
-  for pyversion in $LATEST_3p9 $LATEST_3p10 $LATEST_3p11; do
-    install_macpython $pyversion 11
-    install_virtualenv
-  done
-fi
+for pyversion in $LATEST_3p9 $LATEST_3p10 $LATEST_3p11 $LATEST_3p12 $LATEST_3p13; do
```

Comment:
The previous logic need to be maintained to install the correct versions.

Reply:
reverted to previous logic